ui/src/shared/parsing/getRangeForDygraph.js

```diff
@@ -79,4 +79,18 @@ const getRange = (
   return [min, max]
 }
 
+const parseNumber = bound => {
+  if (bound) {
+    return +bound
+  }
+
+  return null
+}
```

Comment:
I think this should be written as this:

```js
const coerceToNumber = str => str ? +str : null
```

1. Since this is not a parser, that is not what this is doing.
1. There's no indication it is specific to bounds
1. A ternary is perfectly fine for this
1. A one-line conditional block is unnecessarily verbose